libraries/core-react/src/components/TextField/Field.tsx

```diff
@@ -125,13 +122,10 @@ const Adornments = styled.div<AdornmentsType>`
   & div:nth-child(2) {
     margin-left: ${textfield.spacings.left};
   }
-  ${({ multiline, density }) =>
+  margin-top: ${({ theme }) => theme.spacings.top};
+  ${({ multiline }) =>
     multiline && {
       alignSelf: 'start',
-      marginTop:
-        density === 'compact'
-          ? inputToken.modes.compact.spacings.top
-          : textfield.spacings.top,
```

Comment:
Not sure why the `InputToken` is used as compact token here, while textfield token didn't have any compact modes? Added mode to textfield token because that's needed in the `useToken` hook.